test/extensions/filters/http/oauth2/filter_test.cc

```diff
@@ -1838,7 +2076,7 @@ TEST_P(OAuth2Test, OAuthAccessTokenSucessWithTokens_oauth_make_token_cookie_http
       {Http::Headers::get().SetCookie.get(),
        "IdToken=some-id-token;version=1;path=/;Max-Age=600;secure"},
       {Http::Headers::get().SetCookie.get(),
-       "RefreshToken=some-refresh-token;version=1;path=/;Max-Age=600;secure"},
+       "RefreshToken=some-refresh-token;version=1;path=/;Max-Age=600;secure;HttpOnly"},
```

Comment:
is this change related to the patch?

Reply:
Yep, I set cookie with `HttpOnly` attribute. Using without this attribute causes a security problem. There is an [issue](https://github.com/envoyproxy/envoy/issues/31979) to turn on attribute by defaut. Because I have added new code I decided to add cookie with this option by default at once.